pkg/drivers/kic/oci/network_create.go

```diff
@@ -71,30 +68,44 @@ func CreateNetwork(ociBin string, networkName string) (net.IP, error) {
 	if err != nil {
 		klog.Warningf("failed to get mtu information from the %s's default network %q: %v", ociBin, defaultBridgeName, err)
 	}
-	// Rather than iterate through all of the valid subnets, give up at 20 to avoid a lengthy user delay for something that is unlikely to work.
-	// will be like 192.168.49.0/24 ,...,192.168.239.0/24
-	subnet, err := network.FreeSubnet(firstSubnetAddr, 10, 20)
-	if err != nil {
-		klog.Errorf("error while trying to create network: %v", err)
-		return nil, errors.Wrap(err, "un-retryable")
-	}
-	info.gateway, err = tryCreateDockerNetwork(ociBin, subnet.IP, defaultSubnetMask, info.mtu, networkName)
-	if err != nil {
-		return info.gateway, fmt.Errorf("failed to create network after 20 attempts")
+
+	// retry up to 5 times to create container network
+	attempts := 0
+	subnetAddr := firstSubnetAddr
+	for attempts < 5 {
```

Comment:
just `for attemtps := 0; attempts < 5; attempts++` may be? would be shorter and easier to read

Reply:
good point, thank you @ilya-zuyev , i've changed that and also aligned the kvm flow with the container one